plugins/modules/junos_static_routes.py

```diff
@@ -268,6 +454,26 @@
   returned: always
   type: list
   sample: ['command 1', 'command 2', 'command 3']
+rendered:
```

Comment:
could you also update before: and after: sections

Reply:
we dont have before and after for rendred state please check other modules